diff --git a/.gitmodules b/.gitmodules
index 3d19d888..4c0e4740 100644
--- a/.gitmodules
+++ b/.gitmodules
@@ -1,3 +1,3 @@
 [submodule "mocks"]
 	path = mocks
-	url = ../mocks
+	url = https://github.com/iv-org/mocks
diff --git a/src/invidious/users.cr b/src/invidious/users.cr
index b763596b..39ce83a2 100644
--- a/src/invidious/users.cr
+++ b/src/invidious/users.cr
@@ -176,7 +176,7 @@ def get_subscription_feed(user, max_results = 40, page = 1)
         else
           values = "VALUES #{user.watched.map { |id| %(('#{id}')) }.join(",")}"
         end
-        videos = PG_DB.query_all("SELECT * FROM #{view_name} WHERE NOT id = ANY (#{values}) ORDER BY published DESC LIMIT $1 OFFSET $2", limit, offset, as: ChannelVideo)
+        videos = PG_DB.query_all("SELECT * FROM #{view_name} WHERE NOT id = ANY (#{values}) AND NOT title ILIKE '%shorts%' AND views > 0 AND (live_now IS NOT TRUE) AND (length_seconds IS NULL OR length_seconds < 1 OR 61 < length_seconds) AND (NOW() - interval '1 month') < published AND published < (NOW() - interval '1 day') AND (premiere_timestamp IS NULL OR premiere_timestamp < (NOW() - interval '1 day')) ORDER BY published DESC LIMIT $1 OFFSET $2", limit, offset, as: ChannelVideo)
       else
         # Sort subscriptions as normal
 
diff --git a/src/invidious/views/template.ecr b/src/invidious/views/template.ecr
index 77265679..b043f514 100644
--- a/src/invidious/views/template.ecr
+++ b/src/invidious/views/template.ecr
@@ -147,8 +147,7 @@
 
                     <div class="pure-u-1 pure-u-md-1-3">
                         <span>
-                            <i class="icon ion-ios-wallet"></i>
-                            <a href="https://invidious.io/donate/"><%= translate(locale, "footer_donate_page") %></a>
+                          mian
                         </span>
                         <span><%= translate(locale, "Current version: ") %> <%= CURRENT_VERSION %>-<%= CURRENT_COMMIT %> @ <%= CURRENT_BRANCH %></span>
                     </div>
